Different messages be specified for each user

Phishion opened the pull request:
pass alert parameter as a function which accepts token parameter in order to set different message per user

jamaalscarlett commented:
@Phishion This looks good, but it appears that the indentation is off.  And can you update the readme with an example like the badge?

jamaalscarlett reviewed (changes requested):
I think this needs to be indented

Phishion commented:
> I think this needs to be indented

Hello, I have updated the indentation and added the [description](https://github.com/jazzband/django-push-notifications/pull/634/commits/b19e8816773c7683a3f33e1f3b738ce68eeb7c5a) for readme, just contact me if you have any questions

jamaalscarlett commented:
@Phishion Can you add the description to the README.rst file?

Phishion commented:
> @Phishion Can you add the description to the README.rst file?

The latest PR is [here](https://github.com/jazzband/django-push-notifications/pull/653), this one will be closed.
